src/components/avatar/index.js

```diff
@@ -246,17 +279,18 @@ export default class Avatar extends PureBaseComponent {
     } = this.props;
     const hasImage = !_.isUndefined(imageSource);
     const ImageContainer = animate ? AnimatedImage : Image;
+
     if (hasImage) {
       return (
         <ImageContainer
           animate={animate}
-          style={[this.styles.image, imageStyle]}
+          style={[this.getContainerStyle(), {...StyleSheet.absoluteFillObject}, imageStyle]}
```

Comment:
`{...StyleSheet.absoluteFillObject}`
spreading an object inside an empty object is good as just passing the object.

Reply:
Fixed.